.circleci/config.yml

```diff
@@ -362,6 +367,26 @@ workflows:
           requires:
             - prep-build-ts-migration-dashboard
 
+  locales_only:
+    when:
+      matches:
+        pattern: /^l10n_crowdin_action$/
+        value: << pipeline.git.branch >>
+    jobs:
+      - prep-deps
+      - get-changed-files-with-git-diff:
+          <<: *exclude_develop_master_rc
+          requires:
+            - prep-deps
```

Comment:
We can skip this step. The "changed files" data we collect here is used later in the e2e test jobs, but they aren't run in this workflow